kratos/python/add_utilities_to_python.cpp

```diff
@@ -452,6 +452,8 @@ void AddUtilitiesToPython(pybind11::module& m)
     class_<MortarUtilities, typename MortarUtilities::Pointer>(m, "MortarUtilities")
     .def(init<>())
     .def("ComputeNodesMeanNormalModelPart",&MortarUtilities::ComputeNodesMeanNormalModelPart)
+    .def("InvertNormalElements",&MortarUtilities::InvertNormal<PointerVectorSet<Element, IndexedObject>>)
```

Comment:
I would suggest the name `InvertElementNormals`, "NormalElements" sounds weird to me

your call though